src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -126,9 +127,6 @@ private void validateFullRemoveDetails(OverseasEntitySubmissionDto overseasEntit
     }
 
     private void validateFullRegistrationDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
-        validateIsSecureRegister(overseasEntitySubmissionDto.getIsSecureRegister(), true, errors, loggingContext);
```

Comment:
I feel we should have a comment somewhere (perhaps around line 74 in the `validateFull()` method of this class?) just stating that fields used **only** for web navigation purposes are deliberately not validated by the API.

Reply:
Discussed this. Reason is to document the difference in validation behaviour for 'future generations' 😄.

Suggest we add this as the first line in the `validateFullRegistrationDetails()` method:

// Note that model fields used purely to support web journey navigation are not submitted with the filing and therefore aren't checked (e.g. isSecureRegister)

@markpit @lduranteau sound ok?